src/index.html

```diff
@@ -1,12 +1,360 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo"
+          width="40"
+          height="40"
+        >
+      </a>
+      <nav class="header__navigation">
+        <ul class="header__list">
+          <li class="header__item">
+            <a 
+            href="index.html" 
+            class="header__item-link is-active"
+            > 
+              Apple 
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Samsung </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Smartphones </a>
+          </li>
+
+          <li class="header__item">
+            <a 
+            href="index.html" 
+            class="header__item-link" 
+            data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Gadgets </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Tablets </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Photo </a>
+          </li>
+
+          <li class="header__item">
+            <a href="index.html" class="header__item-link"> Video </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="catalog__card card" data-qa="card">
+          <img
+            src="/images/imac.jpeg"
+            alt="Apple A1419 iMac 27 Retina 5K"
+            width="160"
+            height="134"
+            class="card__image"
+          >
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__block">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__review">Reviews: 5</p>
+          </div>
+          <div class="card__block card__block--second">
+            <p class="card__price">Price:</p>
+
+            <p class="card__price-value">$2,199</p>
```

Comment:
```suggestion
            <p class="card__review">
               Reviews: 5
            </p>
          </div>
          
          <div class="card__block card__block--second">
            <p class="card__price">
               Price:
             </p>

            <p class="card__price-value">
              $2,199
            </p>
```
Make your code more readable